test/e2e/workspacetype/controller_test.go

```diff
@@ -287,10 +288,16 @@ func TestClusterWorkspaceTypes(t *testing.T) {
 				})
 
 				t.Logf("Expect workspace to be stuck in initializing phase")
-				time.Sleep(5 * time.Second)
```

Comment:
This is another case where there's a desire to wait a ~short amount of time and make sure an event does not happen. Not sure how we want to approach those.

Reply:
It is not - we are waiting to see it be in initializing, no?

Reply:
My comment about proving negatives and assuming time-scales for eventually consistent systems applies to this case as well.